Models/Article.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Nooter.API.Models
+{
+    public class Article
+    {
+        public Guid Id { get; set; }
+        public string Title { get; set; }
+        public string Body { get; set; }
+
+        public Guid AuthorId { get; set; }

```

Comment:
add `public AppUser Author { get; set; }` property. This will add binding with Users table